features/networking/ovn_ipsec.feature

```diff
@@ -157,3 +157,68 @@ Feature: OVNKubernetes IPsec related networking scenarios
     # Example ESP packet un-encrypted will look like 16:37:16.309297 IP ip-10-0-x-x.us-east-2.compute.internal > ip-10-0-x-x.us-east-2.compute.internal: ESP(spi=0xf50c771c,seq=0xfaad)
     And the output should match: 
        | <%= cb.workers[0].name %>.* > <%= cb.workers[1].name %>.*: ESP |
+       
+  # @author weliang@redhat.com
+  # @case_id OCP-40569
+  @admin
```

Comment:
should this be `@destructive` since this changes the whole network?

Reply:
Yes, it should be, fixed it now, thanks!